composer.json

```diff
@@ -19,7 +19,7 @@
     "illuminate/translation": "5.5.x|5.6.x|5.7.x|5.8.x",
     "symfony/finder": "~3.0|~4.0",
     "tanmuhittin/laravel-google-translate": "^0.11.0",
-    "andrey-helldar/laravel-lang-publisher": "^1.0|^1.5"
+    "andrey-helldar/laravel-lang-publisher": "^1.0"
```

Comment:
@tanmuhittin, hi!
My now bad 😔

Now I installed the package to the project and found my mistake: [caouecs/Laravel-lang](https://github.com/caouecs/Laravel-lang/blob/master/composer.json) does not require a specific version of the Laravel framework, so it will always install version `4.0` for Laravel 5.8...

I made some corrections to my package:
1. Removed `1.5. *` releases and replaced them with `1.6.0` (`^1.0`).
2. Added version `2.0.0` (`^2.0`) for installing to the Laravel 5.8+.

Now version 1.0 contains updates for Laravel 5.3-5.7, and version 2.0 for 5.8 and higher.

Please, update your PR to:
```json
"andrey-helldar/laravel-lang-publisher": "^1.0|^2.0"
```

I apologize again for my mistake 😔